src/NuGet.Indexing/VerifiedPackages.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using FrameworkLogger = Microsoft.Extensions.Logging.ILogger;
+using System.IO;
+
+namespace NuGet.Indexing
+{
+    public static class VerifiedPackages
```

Comment:
I'm torn on this. You're correct that this could be internal, but should it? Only `NuGetSearcherManager` uses this `VerifiedPackages` class today, but this may be a useful helper class in the future. Thoughts?